code/game/objects/items/toys.dm

```diff
@@ -886,6 +1093,140 @@
 	name = "plastic bouquet"
 	desc = "A cheap plastic bouquet of flowers. Smells like cheap, toxic plastic."
 
+//////////////////////////////////////////////////////
+//				Magic 8-Ball / Conch				//
+//////////////////////////////////////////////////////
+
+/obj/item/toy/eight_ball
+	name = "\improper Magic 8-Ball"
+	desc = "Mystical! Magical! Ages 8+!"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "eight-ball"
+	var/use_action = "shakes the ball"
+	var/cooldown = 0
+	var/list/possible_answers = list("Definitely.", "All signs point to yes.", "Most likely.", "Yes.", "Ask again later.", "Better not tell you now.", "Future unclear.", "Maybe.", "Doubtful.", "No.", "Don't count on it.", "Never.")
+
+/obj/item/toy/eight_ball/attack_self(mob/user as mob)
+	if(!cooldown)
+		var/answer = pick(possible_answers)
+		user.visible_message("<span class='notice'>[user] focuses on their question and [use_action]...</span>")
+		user.visible_message("<span class='notice'>The [src] says \"[answer]\"</span>")
+		spawn(30)
+			cooldown = 0
+		return
+
+/obj/item/toy/eight_ball/conch
+	name = "Magic Conch shell"
+	desc = "All hail the Magic Conch!"
+	icon_state = "conch"
+	use_action = "pulls the string"
+	possible_answers = list("Yes.", "No.", "Try asking again.", "Nothing.", "I don't think so.", "Neither.", "Maybe someday.")
+
+// DND Character minis. Use the naming convention (type)character for the icon states.
+/obj/item/toy/character
+	icon = 'icons/obj/toy.dmi'
+	w_class = ITEMSIZE_SMALL
+	pixel_z = 5
+
+/obj/item/toy/character/alien
+	name = "xenomorph xiniature"
+	desc = "A miniature xenomorph. Scary!"
+	icon_state = "aliencharacter"
+/obj/item/toy/character/cleric
+	name = "cleric miniature"
+	desc = "A wee little cleric, with his wee little staff."
+	icon_state = "clericcharacter"
+/obj/item/toy/character/warrior
+	name = "warrior miniature"
+	desc = "That sword would make a decent toothpick."
+	icon_state = "warriorcharacter"
+/obj/item/toy/character/thief
+	name = "thief miniature"
+	desc = "Hey, where did my wallet go!?"
+	icon_state = "thiefcharacter"
+/obj/item/toy/character/wizard
+	name = "wizard miniature"
+	desc = "MAGIC!"
+	icon_state = "wizardcharacter"
+/obj/item/toy/character/cthulhu
+	name = "cthulhu miniature"

```

Comment:
Sorry, despite my comments on the rest, this one SHOULD be capitalized.

Reply:
Not sure about the trademarkyness or status of that anyway, so just changed him to be a 'void one' which is like an 'old one' but with serial numbers filed off